src/main/resources/assets/gregtech/worldgen/overworld/molybdenum_vein.json

```diff
@@ -3,8 +3,10 @@
   "density": 0.5,
```

Comment:
I think we should rename this file to `molybdenite_vein.json` to correctly represent what is generated.

Reply:
I have renamed this file.